frontend/src/BudgetModeDisplay.tsx

```diff
@@ -22,25 +21,22 @@ export function BudgetModeDisplay({
     maxFleetSupply,
     unitCounts,
 }: Props) {
-    return (
-        <>
-            <Typography level="h4">
-                {`Resources Remaining: ${
-                    resourceBudget - getTotalCost(unitCounts)
-                }`}
-            </Typography>
-            <Typography level="h4">
-                {`Capacity Remaining: ${
-                    capacityBudget - getTotalCapacity(unitCounts)
-                }`}
-            </Typography>
-            <Typography level="h4">
-                {`Fleet Supply Remaining: ${getFleetSupplyRemaining(
-                    currentFleetSupply,
-                    maxFleetSupply,
-                    unitCounts
-                )}`}
-            </Typography>
-        </>
-    );
+    return [
+        {
+            label: "Resources Remaining",
+            value: resourceBudget - getTotalCost(unitCounts),
+        },
+        {
+            label: "Capacity Remaining",
+            value: capacityBudget - getTotalCapacity(unitCounts),
+        },
+        {
+            label: "Fleet Supply Remaining",
+            value: getFleetSupplyRemaining(
+                currentFleetSupply,
+                maxFleetSupply,
+                unitCounts
+            ),
+        },
+    ].map(DisplayField);
```

Comment:
I love this more functional and point-free style of composing the components here. Reads really nicely to me.

Reply:
Nice! Same, this is my favorite 🤩 I've heard it's point-free style but didn't know it was a functional programming thing. I should probably just convert smh